cypress/scripts/run-starter-prototype.js

```diff
@@ -0,0 +1,14 @@
+const os = require('os')
+const path = require('path')
+
+const { mkPrototype, startPrototype, installExtensions } = require('../../__tests__/util')
+
+const defaultKitPath = path.join(os.tmpdir(), 'cypress/temp/test-project')
+
+const testDir = path.resolve(process.env.KIT_TEST_DIR || defaultKitPath)
+
+;(async () => {
+  await mkPrototype(testDir, { overwrite: true, allowTracking: false })
+  await installExtensions(testDir, '@govuk-prototype-kit/step-by-step@1')
```

Comment:
Is there a reason we're installing the old version of the step-by-step extension?